commerce-price-list-service/src/main/java/com/liferay/commerce/price/list/internal/upgrade/v3_0_0/CommercePriceListUpgradeProcess.java

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.commerce.price.list.internal.upgrade.v3_0_0;
+
+import com.liferay.commerce.price.list.internal.upgrade.base.BaseCommercePriceListUpgradeProcess;
+import com.liferay.commerce.price.list.model.impl.CommercePriceListImpl;
+import com.liferay.portal.kernel.util.StringUtil;
+
+/**
+ * @author Luca Pellizzon
+ */
+public class CommercePriceListUpgradeProcess
+	extends BaseCommercePriceListUpgradeProcess {
+
+	@Override
+	public void doUpgrade() throws Exception {
+
+		// Add commerceCurrencyCode column
+
+		String newColumnName = "commerceCurrencyCode";
```

Comment:
@luca-pellizzon we can avoid this variable

Reply:
updated